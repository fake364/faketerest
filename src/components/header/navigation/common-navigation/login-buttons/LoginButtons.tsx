import React from 'react';
import PrimaryButton from '../../../../../common/components/buttons/primary-button/PrimaryButton';

type Props = {};

const LoginButtons: React.FC<Props> = () => {
	return (
		<div>
			<PrimaryButton className="mr-[8px]">Login</PrimaryButton>
			<PrimaryButton>Register</PrimaryButton>
		</div>
	);
};

export default LoginButtons;
