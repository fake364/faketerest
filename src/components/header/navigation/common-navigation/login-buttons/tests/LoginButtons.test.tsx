import React from 'react';
import { shallow } from 'enzyme';
import LoginButtons from '../LoginButtons';

describe('LoginButtons', () => {
  it('should render two buttons, one secondary and another primary', () => {
    const wrapper = shallow(<LoginButtons />);
    expect(wrapper).toMatchSnapshot();
  });
});
