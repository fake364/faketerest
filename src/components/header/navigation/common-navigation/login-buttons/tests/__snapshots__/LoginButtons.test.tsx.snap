// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginButtons should render two buttons, one secondary and another primary 1`] = `
<Fragment>
  <PrimaryButton
    className="mr-[8px]"
  >
    loginButtons.login
  </PrimaryButton>
  <SecondaryButton>
    loginButtons.register
  </SecondaryButton>
</Fragment>
`;
